corehq/apps/userreports/tasks.py

```diff
@@ -48,12 +48,15 @@ def _build_indicators(indicator_config_id, relevant_ids):
             # save is a noop if the filter doesn't match
             adapter.save(doc)
             last_id = doc.get('_id')
-            redis_client.srem(redis_key, last_id)
+            try:
+                redis_client.lrem(redis_key, 1, last_id)
+            except:
```

Comment:
nit: sometimes it's nice to figure out what errors we're actually expecting cause in this case if you spelled `lats_id` wrong we'd never know